appengine/flexible/vision/main.py

```diff
@@ -0,0 +1,127 @@
+# Copyright 2015 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START app]
+from datetime import datetime
+import logging
+
+from flask import Flask, redirect, render_template, request
+
+from google.cloud import datastore
+from google.cloud import storage
+from google.cloud import vision
+
+
+CLOUD_STORAGE_BUCKET = '<your-storage-bucket>'
```

Comment:
Use os.environ and env_variables in app.yaml? (or do you have a good reason for this?)